src/command-line/add.js

```diff
@@ -1,15 +1,15 @@
 var ClientManager = new require("../clientManager");
 var bcrypt = require("bcrypt-nodejs");
 var program = require("commander");
-var mkdirp = require("mkdirp");
+var fsextra = require("fs-extra");
 var Helper = require("../helper");
 
 program
 	.command("add <name>")
 	.description("Add a new user")
 	.action(function(name/* , password */) {
 		try {
-			mkdirp.sync(Helper.USERS_PATH);
+			fsextra.ensureDirSync(Helper.USERS_PATH);
```

Comment:
This is ensured in `index.js`, so there's no need to do it again in the add command (remove the try/catch completely).